exotations/exotica_core_task_maps/src/eff_box.cpp

```diff
@@ -118,6 +118,11 @@ void EffBox::PublishObjectsAsMarkerArray()
     pub_markers_.publish(ma);
 }
 
+void EffBox::AssignScene(ScenePtr scene)
+{
+    scene_ = scene;
```

Comment:
Shouldn't this be automatically done or are you missing the `Initialize` call here?

Reply:
Ahh i thought to use `scene_` you had to override `AssignScene`, resolved in latest commit.